python/ppml/src/bigdl/ppml/ppml_context.py

```diff
@@ -38,34 +38,32 @@ def __init__(self, app_name, ppml_args=None, spark_conf=None):
             for (k, v) in spark_conf.getAll():
                 conf[k] = v
         if ppml_args:
-            kms_type = ppml_args.get("kms_type", "SimpleKeyManagementService")
-            conf["spark.bigdl.kms.type"] = kms_type
+            kms_type = ppml_args.get("kms_type", "")
+            conf["spark.bigdl.primaryKey.defaultKey.kms.type"] = kms_type
```

Comment:
why hardcode defaultKey？

Reply:
defaultKey is a name for python user who calls single-key argument wrapper.